hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/dto/AttributeSearchRequestJsonTest.kt

```diff
@@ -18,7 +26,7 @@ class AttributeSearchRequestTest(@Autowired val objectMapper: ObjectMapper) {
             "matchers": [
               {
                 "joinType": "AND",
-                "textMatchers": [
+                "stringMatchers": [
```

Comment:
Renamed the text matcher to match the type it supports to follow same convention applied to other types